modules/search/instant-search/components/api.js

```diff
@@ -3,26 +3,73 @@
  */
 import fetch from 'unfetch';
 
-const FIELDS = [
-	'author',
-	'comment_count',
-	'date',
-	'excerpt_html',
-	'gravatar_url',
-	'permalink.url.raw',
-	'title_html',
-];
+/**
+ * @preserve jquery-param (c) 2015 KNOWLEDGECODE | MIT
+ * @param {object} a - parameters to add
+ * @returns {string} url query string
+ * From https://github.com/knowledgecode/jquery-param
+ */
+function query_params( a ) {
+	//eslint-disable-next-line no-var
```

Comment:
What's the reasoning behind the mix of es5 and es6 in this file? Would it be possible to stick to es6, since the file will be transpiled?